packages/database/src/migrations/20201110234830-DeleteOldLaosSchoolsTextbookData-modifies-data.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    -- Drop triggers so that all the deletes don't flood the dhis sync queue
```

Comment:
```suggestion
    -- Drop triggers so that change notifications don't slow the migration down to the point it fails
    -- This is safe because all data is internal, so no change handlers need to handle this for external sync queues
```